dashboard/app/models/rubric_ai_evaluation.rb

```diff
@@ -25,7 +25,7 @@ class RubricAiEvaluation < ApplicationRecord
   belongs_to :requester, class_name: 'User'
   belongs_to :rubric
 
-  has_many :learning_goal_ai_evaluations, inverse_of: :rubric_ai_evaluation
+  has_many :learning_goal_ai_evaluations, inverse_of: :rubric_ai_evaluation, dependent: :destroy
```

Comment:
What does this do here?

Reply:
oops, I should definitely have explained this! this makes it so that, during local development, when you do `RubricAiEvaluation.find(...).destroy`, instead of getting a foreign key error and having to go delete the corresponding `learning_goal_ai_evaluations` manually, it will delete them for you. I ended up doing this a fair amount during development and thought it would be worth keeping this change in for future use. FYI @Nokondi 

we do not appear to have any codepaths for deleting RubricAiEvaluation objects in our product, so this should have no effect for end users.